projects/packages/import/src/endpoints/trait-import.php

```diff
@@ -12,6 +12,11 @@
 /**
  * Import trait. Base class for all import endpoints.
  */
+/**
+ * Batching details.
+ *
+ * @property array|false $allow_batch
```

Comment:
I'm curious why you decided not to actually include that property into code.

Reply:
Putting it in code made Phan happy, but broke PHP:

> If a trait defines a property then a class can not define a property with the same name unless it is compatible (same visibility and type, readonly modifier, and **initial value**)

https://www.php.net/manual/en/language.oop5.traits.php#language.oop5.traits.properties

The various classes that inherited this trait set their own varied initial values, disqualifying the trait from setting one despite having a function that calls said property.

See also: p1739911318263949-slack-C05Q5HSS013